packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -284,6 +342,7 @@ const BulkActionButtons = ({
             onToggleDialog={togglePublishDialog}
             isConfirmButtonLoading={isConfirmButtonLoading}
             onConfirm={handleBulkPublish}
+            slug={slug}
```

Comment:
How come you feel it's best to pass it as a prop when we can use the selector in the component?

Reply:
removed